copilot-bootstrap.py

```diff
@@ -129,16 +129,16 @@ def setup_templates():
             "public-manifest": templateEnv.get_template("svc/manifest-public.yml"),
             "backend-manifest": templateEnv.get_template("svc/manifest-backend.yml"),
             "opensearch": templateEnv.get_template("svc/addons/opensearch.yml"),
-            "postgres": templateEnv.get_template("svc/addons/postgres.yml"),
+            "rds-pg": templateEnv.get_template("svc/addons/rds-pg.yml"),
```

Comment:
We will still need the other `aurora` Postgres template.

```suggestion
            "aurora-pg": templateEnv.get_template("svc/addons/aurora-pg.yml"),
            "rds-pg": templateEnv.get_template("svc/addons/rds-pg.yml"),
```